packages/web-app-progress-bars/tests/e2e/progressBar.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, Page, expect } from '@playwright/test'
+import { AccountPage } from '../../../../support/pages/accountPage'
+import { loginAsUser, logout } from '../../../../support/helpers/authHelper'
+
+let adminPage: Page
+
+test.beforeEach(async ({ browser }) => {
+  const admin = await loginAsUser(browser, 'admin', 'admin')
+  adminPage = admin.page
+})
+
+test.afterEach(async () => {
+  await logout(adminPage)
+})
+
+test('select the progressBarOption from the web-app-progress-bars extension', async () => {
```

Comment:
```suggestion
test('select the progressBarOption on the account page', async () => {
```
this only tests if the dropdown shows the Nyan Cat option and if its selectable, nothing more